recipe/fix-install_comic_fonts.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/Code/GraphMol/MolDraw2D/CMakeLists.txt b/Code/GraphMol/MolDraw2D/CMakeLists.txt
```

Comment:
From the comment, seems like this patch came from upstream? Could you include a link to the commit if so?

Reply:
Sure, I added a comment to the recipe. It's part of a larger release commit: https://github.com/rdkit/rdkit/commit/ba0b6f3bafb391c17d7da9783f09817a4ba0f8f1